hammer_cli_foreman_discovery.gemspec

```diff
@@ -24,4 +23,5 @@ DESC
   s.add_dependency 'hammer_cli_foreman', '~> 3.10'
 
   s.required_ruby_version = '>= 2.7', '< 4'
+  s.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
If everyone has MFA set up, that's ok but know that it's incompatible with https://github.com/theforeman/actions?tab=readme-ov-file#gem-release